node/worker/worker.go

```diff
@@ -315,15 +320,11 @@ func (w *Worker) SuperCommitteeForNextEpoch(
 					beaconEpoch, beacon,
```

Comment:
Is it possible that different validator has different speed, their database are not synced. Then their proposed epoch will be different

Reply:
resolved offline.